browser/components/utils/ToggleableSection.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import Button from '../atoms/Button/index'
+import Icon from '../atoms/Icon/index'
+import './ToggleableSection.sass'
+
+export default class ToggleableSection extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      open: this.constructor.initialState
```

Comment:
you need to defined this if you're going to access it like this. Add

```js
static initialState = true
```
above the constructor